addon/components/citation-widget/template.hbs

```diff
@@ -1,8 +1,24 @@
-<div class="citation-widget">
-    <div class="f-w-xl">APA</div>
-    <span>{{apa}} </span>
-    <div class="f-w-xl m-t-md">MLA</div>
-    <span> {{mla}} </span>
-    <div class="f-w-xl m-t-md">Chicago</div>
-    <span> {{chicago}} </span>
+<div>
+    <div id="citationList" class="m-b-md">
+        <div class="f-w-xl">APA</div>
+        <span>{{apa}} </span>
```

Comment:
Should maybe add a space in front of the APA citation to keep it consistent with the other styles